app-foundation/4-projects/development/boa_secret.tf

```diff
@@ -31,32 +31,31 @@ locals {
       "roles/iam.roleAdmin",
       "roles/binaryauthorization.policyEditor"
     ],
-    "${module.boa_ops_project.project_id}" = [
```

Comment:
Same comment as above, can't we keep this as it?

Reply:
resolved offline - keeping as is due to `The "for_each" value depends on resource attributes that cannot be determined
until apply`